SPECS/swig/swig.spec

```diff
@@ -55,7 +55,7 @@ mkdir -p %{buildroot}%{_libdir}/ccache
 ln -fs ../../bin/ccache-swig %{buildroot}%{_libdir}/ccache/swig
 
 %check
-%make_build check PY3=y
+make check PY3=1
```

Comment:
Was the removal of `%make_build` necessary? It allows us to control the build behaviour globally across specs with the macro's definition. Also, it usually speeds the build up by applying the `-j` flag by default.

Reply:
Yeah, it wasn't, you are right, switched it back

Reply:
Buddy build it successful: [550747](https://dev.azure.com/mariner-org/mariner/_build/results?buildId=550747&view=results)